extension/popup.html

```diff
@@ -31,6 +31,7 @@
       -
       - [1]: https://developer.chrome.com/extensions/contentSecurityPolicy
      -->
+    <script src="jquery-3.1.1.min.js"></script>
```

Comment:
It's actually doing a fair bit of work in checking if the DOM is loaded and also getting any clicked anchor. 

Also, since jQuery isn't getting loaded with every request(since it's an extension rather than a website), I don't think the size is a big issue.

But if you know the JS off hand to do handle checking the anchor click events, I'll drop jQuery and put that in instead.